.setup/bin/sample_courses/models/course/course_create.py

```diff
@@ -161,25 +160,20 @@ def create(self):
         self.electronic_gradeable_version = Table("electronic_gradeable_version", self.metadata, autoload=True)
         for gradeable in self.gradeables:
             gradeable.create(self.conn, self.gradeable_table, self.electronic_table, self.peer_assign, reg_table, self.component_table, self.mark_table)
-            form = os.path.join(self.course_path, "config", "form", "form_{}.json".format(gradeable.id))
+            form = os.path.join(self.course_path, "config", "form", f"form_{gradeable.id}.json")
             with open(form, "w") as open_file:
                 json.dump(gradeable.create_form(), open_file, indent=2)
-        os.system("chown -f submitty_php:{}_tas_www {}".format(self.code, os.path.join(self.course_path, "config", "form", "*")))
+        os.system(f"chown -f submitty_php:{self.code}_tas_www {os.path.join(self.course_path, 'config', 'form', '*')}")
```

Comment:
Lines like this should be split into multiple lines in future refactors.  We probably want to minimize the number of inline function calls in f-strings.